src/components/Drawer.js

```diff
@@ -152,11 +159,11 @@ const Drawer = React.createClass({
         backgroundColor: StyleConstants.Colors.PORCELAIN,
         boxShadow: StyleConstants.ShadowHigh,
 
-        '@media (max-width: 750px)': {
+        [`@media (max-width: ${this.props.breakPoints.medium}px)`]: {
```

Comment:
Thank you @iheartrachie! The bracket syntax is called _Computed property names_ and is new in ES2015.

See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Computed_property_names

We're doing something similar in raja, take a look at `Icon` in `serenity/constants/Category.js`.